Source/API/CBLRevision.m

```diff
@@ -308,6 +308,20 @@ - (CBLSavedRevision*) parentRevision {
     return _parentRevID ? [_document revisionWithID: _parentRevID] : nil;
 }
 
+- (void) setProperties: (NSMutableDictionary*)properties {
+    @synchronized(self) { // Atomic property
```

Comment:
Doesn't actually need to be atomic since our API is explicitly not thread-safe, so you can take out the `@synchronized`...